lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -91,13 +91,13 @@ def purchase(money, payment, options = {})
       def capture(money, authorization, options = {})
         post = {}
 
-        add_amount(post, money, options)
         add_application_fee(post, options)
 
         if emv_tc_response = options.delete(:icc_data)
           post[:card] = { emv_approval_data: emv_tc_response }
           commit(:post, "charges/#{CGI.escape(authorization)}", post, options)
         else
+          add_amount(post, money, options)
```

Comment:
I'm having total deja vu here - didn't I ask this exact question about this exact line somewhere? Is this real life? THERE IS NO SPOON